src/etherscan/EtherscanResponse.ts

```diff
@@ -0,0 +1,18 @@
+export default class EtherscanResponse {
+  public readonly status: number;
+
+  public readonly message: string;
+
+  public constructor(response: any) {
```

Comment:
I'd rather have something like `type EtherscanResponse = any` instead of a direct `any`. This way it'd be easier to change in the future.

Reply:
Hmm not sure how to proceed on this one. It's common practice to have constructor with any type when casting service response to some model. Not sure what would be the benefit of defining type since every http library will return any type s it ill never be detected at compile time though.

I'm doing some transformation here because status isn't number for some reason, and message prop contains useless OK and NOTOK :-1:

Reply:
I got your point @mpetrunic. This is fine. Sorry for the misunderstanding.